src/main/webapp/index.jsp

```diff
@@ -51,3 +53,5 @@
   </div>
 </body>
 </html>
+
+<% System.out.println("STATS: index.jsp: " + (System.currentTimeMillis() - startTime) + "ms"); %>
```

Comment:
You might also want to look into logging rather than using System.out.  The rule of thumb I've heard is that programs generally should use loggers rather than stdout for everything except for main program output which, in a web server is nothing.

Reply:
I thought there might be some special method I should call, but I found System.out being used in other parts of the code already, so I decided to just use it. What do you think I should do then?